package.json

```diff
@@ -37,7 +37,7 @@
         "test-version-tolerance": "mocha -r ts-node/register --timeout 2000 './test/version-tolerance/**/!(sampleTest).spec.ts'",
         "clone:specs": "git clone https://github.com/Azure/azure-rest-api-specs.git ./.tmp/specs",
         "smoke-test": "ts-node ./test/utils/smoke-test.ts",
-        "copyFiles": "copyfiles src/**/*.hbs dist/ && copyfiles src/**/**/*.hbs dist/"
+        "copyFiles": "cp src/**/*.hbs dist/ && cp src/**/**/*.hbs dist/ && cp src/generators/samples/*.hbs dist/src/generators/samples/ && cp src/generators/static/*.hbs dist/src/generators/static/ && cp src/generators/test/*.hbs dist/src/generators/test/ && cp src/restLevelClient/*.hbs dist/src/restLevelClient/"
```

Comment:
Does this work cross platform?

Reply:
cp normally works for linux/mac boxes. I use a windows machine and it is working fine for that too

Reply:
I think it works in PowerShell but it doesn't work in windows cmd 😢 
![image](https://user-images.githubusercontent.com/21109913/164324857-46222025-3a8b-4ae9-9f52-7376841d421d.png)

Is there other package we can use that works cross platform? Or can we write a node script that does this for us?

Reply:
I did not use the powershell. I used it in the regular command prompt

Reply:
Awesome! Thanks for verifying